darkdraw/ansihtml.py

```diff
@@ -96,6 +96,7 @@ def save_ansihtml(vd, p, *sheets):
 
         dwg._scr = mock.MagicMock(__bool__=mock.Mock(return_value=True),
                                   getmaxyx=mock.Mock(return_value=(9999, 9999)))
+        dwg.reload()
```

Comment:
This is not good; what if I made changes to the drawing?  Then this reload will revert those changes as it reloads what's on disk.

At most it should be `ensureLoaded()`.

Reply:
I thought that, too! I actually tested it out, and it surprisingly did not revert the changes. I think because this is a fresh `Drawing()`? But, I will think of a better approach.

Reply:
`ensureLoaded` will not work, because it checks if a Sheet is loaded by checking its rows! `Drawing` does not set its rows through `reload()`. Its rows come from the source in its instantiation.

```
@property
def rows(self):
    return self.source.rows
```

So they come from the already loaded `vs`. And also, calling a `reload()` on a Drawing will not actually force-reload its source.

```
def reload(self):
    self.source.ensureLoaded()
    vd.sync()
    self.minX, self.minY, self.maxX, self.maxY = boundingBox(self.source.rows)
    if self._scr:
        self.draw(self._scr)
```

That is why calling the reload did not change what was actually drawn. So, I actually think calling `reload` here is the right approach.